src/VisualStudio/Core/Def/Options/PackageSettingsPersister.cs

```diff
@@ -17,15 +16,11 @@ namespace Microsoft.VisualStudio.LanguageServices.Options;
 internal sealed class PackageSettingsPersister : IOptionPersister
```

Comment:
not sure if we need this type at all now.

Reply:
It sure doesn't look like it to me